back/src/controllers/userController.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable quotes */
+import { userService } from "../services/userService.js";
+
+async function getAll(req, res, next) {
+  const users = await userService.getAllActive();
+
+  res.send(users.map(userService.normalize));
+}
+
+async function updateName(req, res, next) {
+  try {
+    const { email, fullName } = req.body;
+
+    await userService.updateName(email, fullName);
+
+    res.send({ status: "OK" });
+  } catch (error) {
+    res.send({ status: "FAILED" });
```

Comment:
Consider setting statuses by express, as in the comment above. If you want to add a specific message you can add a send an object with a message field

Reply:
Fixed. I was trying to complete the task ASAP, so decided to get a short way))